test/functional/connection.test.js

```diff
@@ -70,6 +71,21 @@ describe('Connection - functional', function () {
     }
   });
 
+  it('should only pass one argument (topology and not error) for topology "open" events', function (done) {
+    var configuration = this.configuration;
+    var client = configuration.newClient({ w: 1 }, { maxPoolSize: 1 });
```

Comment:
```suggestion
    const configuration = this.configuration;
    const client = configuration.newClient({ w: 1 }, { maxPoolSize: 1 });
```

We tend to modernize code as we come across it, I know you were going off the nearby test but its nice to try and update things as we get to them.

Reply:
Makes sense. Thanks!